cmd/run.go

```diff
@@ -71,6 +71,7 @@ func bindRunFlags(cmd *cobra.Command, config *app.Config) {
 	cmd.Flags().StringVar(&config.JaegerService, "jaeger-service", "charon", "Service name used for jaeger tracing.")
 	cmd.Flags().BoolVar(&config.SimnetBMock, "simnet-beacon-mock", false, "Enables an internal mock beacon node for running a simnet.")
 	cmd.Flags().BoolVar(&config.SimnetVMock, "simnet-validator-mock", false, "Enables an internal mock validator client when running a simnet. Requires simnet-beacon-mock.")
+	cmd.Flags().StringVar(&config.SimnetValidatorKeys, "simnet-validator-keys", ".charon/validator_keys", "The directory containing the simnet validator key shares.")
```

Comment:
Lets change the flag and config variable to add a "dir" suffix, `simnet-validator-keys-dir`